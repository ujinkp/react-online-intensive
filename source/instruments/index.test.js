//Core
import { sum } from './';

test('sum function should be a function', () =>{
    expect(sum).toBeInstanceOf(Function);
})

test('sum function should be throw, when called with non-number type as second argument', () =>{
    expect(() => sum(2, 'привет')).toThrow();
})
