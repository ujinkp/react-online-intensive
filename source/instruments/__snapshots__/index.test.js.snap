// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`instruments: sum function should return an additionof two arguments passed 1`] = `9`;
